File: Python/reverse_letter.py
def reverse_letter(string):
    string = string.lower()

    alphabet = "abcdefghijklmnopqrstuvwxyz"

    temp_string = ""

    for char in string:
        if (char in alphabet):
            temp_string = char + temp_string
        else:
            continue

    return temp_string

File: CSharp/ReverseLetter.cs
namespace myjinxin
{
    using System;
    
    public class Kata
    {
        public string ReverseLetter(string str)
        {
            str = str.ToLower();

            string alphabet = "abcdefghijklmnopqrstuvwxyz";

            string resultString = "";
            
            for (int i = 0; i < str.Length; i++)
            {
                if (alphabet.Contains(str[i]))
                {
                    resultString = str[i] + resultString;
                } else
                {
                    continue;
                }
            }

            return resultString;
        }
    }
}